src/components/Dropdown/components/menu/menu.jsx

```diff
@@ -17,20 +17,22 @@ const Menu = ({ children, Renderer, selectProps, dropdownMenuWrapperClassName, .
   const withoutAnimation = true;
 
   return (
-    <components.Menu
-      {...props}
-      className={cx(
-        styles.dropdownMenuWrapper,
-        {
-          [styles.dropdownMenuWrapperFixedPosition]: withFixedPosition,
-          [styles.withoutAnimation]: withoutAnimation
-        },
-        dropdownMenuWrapperClassName
-      )}
-    >
-      {Renderer && Renderer(rendererProps)}
-      {!Renderer && children}
-    </components.Menu>
+    <div id={id} role="listbox" aria-label={ariaLabel}>
```

Comment:
@shaharzil managed to upgrade to latest react-select version which already supports it out-of-the-box, let's sync about it, it might save us these changes

Reply:
Hi @or-nuri-monday, I played with the react-select library and found this prop (called innerProp) to add props to the menu wrapper:
<img width="390" alt="image" src="https://github.com/mondaycom/monday-ui-react-core/assets/33456543/12e95a3c-11e7-47f7-a311-b04f834f96b8">

Can you please try it and let us know if it solves the issue? 🙏